pg-client.cabal

```diff
@@ -24,6 +24,9 @@ custom-setup
     , Cabal  >=1.10
 
 library
+  extra-libraries:
+      krb5
+    , ssl
```

Comment:
Should we cherry pick this off into its own PR? These C libraries are required, and `haskell.nix` uses this stanza to figure out what C libs to automatically pull in from nixpkgs, but it seems like we should be calling them out here regardless.